private-templates-service/client/src/components/Common/DeleteModal/styled.tsx

```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+export const ConfirmationDescription = styled.div`
+  margin-bottom: 0px;
```

Comment:
Not needed, `div` automatically has `margin-bottom: 0px` i believe

Reply:
Fixed!